resources-generator/src/main/kotlin/dev/icerock/gradle/generator/common/CommonAssetsGenerator.kt

```diff
@@ -0,0 +1,12 @@
+package dev.icerock.gradle.generator.common
+
+import dev.icerock.gradle.generator.AssetsGenerator
+import org.gradle.api.file.FileTree
+import java.io.File
+
+class CommonAssetsGenerator(inputFile: FileTree) : AssetsGenerator(inputFile) {
```

Comment:
we should generate MR accessors in for assets too. Get by name should not be single way to work with assets - we should have both ways as in Images. It can be implemented with `ResourcesContainer` as in Images